catalog/core/catalog-core-standardframework/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -322,11 +321,23 @@
     <bean id="reliableResourceDownloadManager"
           class="ddf.catalog.resource.download.ReliableResourceDownloadManager"
           init-method="init" destroy-method="cleanUp">
-        <argument ref="deprecatedProductCache"/>
-        <argument ref="retrieveStatusEventPublisher"/>
-        <argument ref="retrieveStatusEventListener"/>
-        <argument ref="downloadStatusInfo"/>
-        <argument ref="downloadThreadPool"/>
+        <cm:managed-properties persistent-id="ddf.catalog.resource.download.ReliableResourceDownloadManager"
```

Comment:
If we create a new `persistent-id` for this, we will break the export/import of the configurations from 2.9.x to 2.10.x. Should we do this now or wait until we have all the export/import capabilities to do the conversion?

If we decide to go ahead with this, we probably want to create a new `metatype.xml` file for the properties that were removed from the catalog framework's `metatype.xml` file. We may also want to add something to our release notes related to this so people know the configurations need to be manually transferred.